### ISSUE TITLE
Module not found in nextjs app

### ISSUE THREAD

**Issue by nrgapple** [spencermountain/compromise#908]
```
Module not found: Can't resolve 'compromise/three'
   7 | import model from 'wink-eng-lite-model';
   8 | 
>  9 | import nlpc from 'compromise/three';
  10 | 
  11 | const nlp = winkNLP(model);

```

**Comment by spencermountain** [comment]
hey Adam, are you on >= v14?
i don't know anything about how nextjs handles named exports, but `from 'compromise'` should load /three by default, too
cheers

**Comment by spencermountain** [comment]
should be fixed by 14.1.0
cheers
